src/components/IstioWizards/Slider/Slider.tsx

```diff
@@ -60,6 +68,11 @@ class Slider extends React.Component<Props, State> {
     };
   }
 
+  componentDidMount() {
+    // This empty setState forces a re-render which resolves an issue with initial tick_label placement
```

Comment:
Why? What exactly is the issue here?

Reply:
I'm not sure what else to say, without the re-render prior to initial display the tick label placement was incorrect.